src/ape/api/query.py

```diff
@@ -12,6 +12,7 @@
     "AccountTransactionQuery",
     "ContractEventQuery",
     "ContractMethodQuery",
+    "LogFilter",  # type: ignore # noqa: F821
```

Comment:
What does this do?
Since it is not defined anywhere, does this actually work?

Reply:
I think it might have been something that banteg defined, and it should be using `ContractEventQuery` instead

Reply:
Do we need this line of code though?

Reply:
I think not, I think it was copy-paste from Banteg's POC and we need to re-align to using `ContractEventQuery` instead